Makefile

```diff
@@ -2,7 +2,7 @@ COMMIT?=$(shell git describe HEAD)$(shell git diff --quiet || echo '+dirty')
 
 # Use linker flags to provide commit info
 VERSION_LDFLAGS=-X=github.com/foundriesio/fioctl/subcommands/version.Commit=$(COMMIT)
-COMMON_LDFLAGS=-v -linkmode=external $(VERSION_LDFLAGS)
+COMMON_LDFLAGS=-v -s -w -linkmode=external $(VERSION_LDFLAGS)
```

Comment:
What if a developer would like to debug `fioctl`? Maybe those parameters should be set conditionally depending on "build type"?

Reply:
This is a kind of an output the developer gets in case of an (intentional) panic:
```
panic: Something went wrong

goroutine 1 [running]:
github.com/foundriesio/fioctl/subcommands/keys.doShowCA(0xc000436800?, {0x7fbd9fef6609?, 0x4?, 0x7fbd9fef660d?})
	/source/subcommands/keys/ca_show.go:94 +0x4ec
github.com/spf13/cobra.(*Command).execute(0xc000439200, {0xc0001c9f20, 0x2, 0x2})
	/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:920 +0x863
github.com/spf13/cobra.(*Command).ExecuteC(0x7fbda0674020)
	/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:1044 +0x3a5
github.com/spf13/cobra.(*Command).Execute(...)
	/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:968
github.com/foundriesio/fioctl/cmd.Execute()
	/source/cmd/root.go:65 +0x17b
main.main()
	/source/main.go:6 +0xf
```

In 99.9% of cases it is more than enough to get the idea what went wrong.
The symbols being stripped are the ones needed by e.g. `gbd` to debug local variables etc etc.
If the developer _really needs_ them, and know how to work with `gdb`, they would know how to update the Makefile.

For a regular use case we don't need that functionality, while the storage saving is worthy (~20MB on Windows and 10MB on Linux per one binary).

Reply:
The only thing I want to add is if we ever want to use this in a recipe in bitbake we either need to patch the `-s -w` out or just use the [INHIBIT_PACKAGE_STRIP](https://docs.yoctoproject.org/ref-manual/variables.html#term-INHIBIT_PACKAGE_STRIP) and [INHIBIT_SYSROOT_STRIP](https://docs.yoctoproject.org/ref-manual/variables.html#term-INHIBIT_SYSROOT_STRIP) :+1:

Reply:
I think for the Yocto we'd need a separate recipe somewhere in meta-lmp.

Reply:
I mean, it will not use this make file at all, and use it's own built tool chain, like e.g. fioconfig does.

Reply:
Don't worry about developer mode. If you want, `go run ..`. And tools like `delve` basically do that also.